x-pack/plugins/security_solution/public/cases/components/user_action_tree/user_action_username.tsx

```diff
@@ -8,19 +8,22 @@ import React, { memo } from 'react';
 import { EuiToolTip } from '@elastic/eui';
 import { isEmpty } from 'lodash/fp';
 
+import * as i18n from './translations';
+
 interface UserActionUsernameProps {
-  username: string;
-  fullName?: string;
+  username?: string | null;
```

Comment:
is it true that  `username` will only be undefined when `type: alert`?? can we check for this and say something like `Automated Alert` (maybe ask product) instead of `Unknown`?

Reply:
A non authenticated user can create cases. In this case, the username will be `null`. I agree about asking product about a suitable name.

Reply:
maybe user is Kibana?